app/services/data/update-omic-workload-points-calculation.js

```diff
@@ -12,11 +12,11 @@ module.exports = function (workloadReportId, workloadPointsId, t2aWorkloadPoints
       licence_points: licensePoints,
       sdr_points: sdrPoints,
       sdr_conversion_points: sdrPointsConversion,
-      paroms_points: paromsPoints,
```

Comment:
How come these are not just removed.. there might be a good reason but the rest of the PR just removes all references?!

Reply:
they're mandatory columns in the db, thought not to drop the columns from the db for now